.eslintrc.json

```diff
@@ -34,8 +34,8 @@
       // which has convulated return types
       "files": ["./src/index.ts", "./src/utilities/copyToClipboard.ts"],
       "rules": {
-        "@typescript-eslint/explicit-function-return-type": "off",
-        "@typescript-eslint/no-empty-function": "off"
+        "@typescript-eslint/explicit-function-return-type": "error",
+        "@typescript-eslint/no-empty-function": "error"
       }
```

Comment:
Why have you turned on  linter for this ?

Reply:
Respected sir,  Actually I was checking certain lint errors. Let me revert those changes. Thank you.